jaeger-core/src/test/java/com/uber/jaeger/samplers/GuaranteedThroughputSamplerTest.java

```diff
@@ -63,4 +65,27 @@ public void testProbabilityTagsOverrideRateLimitingTags() throws Exception {
     assertEquals(tags.get(Constants.SAMPLER_TYPE_TAG_KEY), ProbabilisticSampler.TYPE);
     assertEquals(tags.get(Constants.SAMPLER_PARAM_TAG_KEY), 0.999);
   }
+
+  @Test
+  public void testUpdate() {
```

Comment:
The tests would be a lot more readable if the test cases are separated into something like  `testUpdate_probabilisticSampler, testUpdate_rateLimitingSampler`, etc